src/runtime/actor/util.rs

```diff
@@ -47,3 +48,33 @@ where
         Ok(())
     }
 }
+
+/// An event which will spawn a supervised actor
+#[derive(Debug)]
+pub struct SpawnActor<A: Actor> {
```

Comment:
Suggestion: `SpawnedActor` because when we handle that event it already happened, right?

Reply:
No, the event will spawn the actor when it is processed.

Reply:
Alright 👌 

And the event is fully pre-determined, meaning no other/user-defined code can be attached to it, such that it would be become more than just an actor spawner?